demo/apisix/conf/apisix.yaml

```diff
@@ -6,6 +6,78 @@ routes:
       nodes:
       type: roundrobin
     plugin_config_id: 1
+  -
+    id: jmap
+    uri: /oidc/jmap
+    service_id: jmap_service_oidc
+    methods:
+      - POST
+    plugins:
+      proxy-rewrite:
+        uri: /jmap
+  -
+    id: jmap_session
+    uri: /oidc/jmap/session
+    service_id: jmap_service_oidc
+    methods:
+      - GET
+    plugins:
+      proxy-rewrite:
+        uri: /jmap/session
+  -
+    id: download
+    uri: /download/*
+    service_id: jmap_service_oidc
+    methods:
+      - GET
+      - OPTIONS
```

Comment:
The OPTIONS request does not need authentication (as current declaration), no?

Anyway I have a ticket handling this OPTIONS, you can just remove it for now